pex/venv/virtualenv.py

```diff
@@ -130,6 +137,7 @@ def create(
         interpreter=None,  # type: Optional[PythonInterpreter]
         force=False,  # type: bool
         copies=False,  # type: bool
+        system_site_packages=False,  # type: bool
```

Comment:
When are system_site_packages relevant if we are ignoring them above in find_site_packages_dir?

Reply:
There is a pre-existing test that uses this new functionality (previously a pyenv-based utility in `pex/testing.py` was used to builr venvs with `--system-site-packages`). That test is an important test that fixed a real bug found in the wild. So just to support that - its not used by production code and I think never will be. Pex is all about hermetic and `--system-site-packages` is all about not!